Makefile

```diff
@@ -208,12 +208,18 @@ GOBIN=$(PROJECT_DIR)/bin go install $(2) ;\
 }
 endef
 
+OPERATOR_SDK = $(shell pwd)/bin/operator-sdk
+.PHONY: operator-sdk
+operator-sdk:
+	curl -Lo ${OPERATOR_SDK} 'https://github.com/operator-framework/operator-sdk/releases/download/v1.25.2/operator-sdk_linux_amd64'
+	chmod +x ${OPERATOR_SDK}
```

Comment:
This makes it always download the binary, doesn't it? It should skip the download if the binary is already in `bin`.

Reply:
Thanks. Fixed.